src/mds/CDir.cc

```diff
@@ -2205,7 +2205,7 @@ void CDir::_omap_commit(int op_prio)
 
       // don't create new dirfrag blindly
       if (!is_new() && !state_test(CDir::STATE_FRAGMENTING))
-	op.stat(NULL, (ceph::real_time*) NULL, NULL);
+	op.stat(NULL, (ceph::real_time*) NULL, (int*)NULL);
```

Comment:
I have a personal preference for C++ style casts, YYMV.

Reply:
Just added nullptr_t overloads to get rid of these.